prov/shm/src/smr_util.h

```diff
@@ -177,7 +177,8 @@ struct smr_addr {
 struct smr_peer_data {
 	struct smr_addr		addr;
 	uint32_t		sar_status;
-	uint32_t		name_sent;
+	uint16_t		name_sent;
+	uint16_t		ipc_invalid;
```

Comment:
You'll need to add a version bump for these changes since you've touched the shm region and protocols